docs/source/Synthesis.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Synthesis of Clifford Circuits\n",
+    "\n",
+    "Executing quantum circuits on a quantum computer requires compilation to representations that conform to all restrictions imposed by the device.\n",
+    "Due to device's limited coherence times and gate fidelity, the compilation process has to be optimized as much as possible.\n",
+    "To this end, an algorithm's description first has to be *synthesized* using the device's gate library.\n",
+    "In addition, circuits have to be *mapped* to the target quantum device to satisfy its connectivity constraints.\n",
+    "Even though Clifford circuits form a finite subgroup of all quantum circuits -- one that is not even universal for quantum computing -- the search space for these problems grows exponentially with respect to the number of considered qubits.\n",
+    "\n",
+    "The *Clifford synthesis approach* in QMAP can be used to produce optimal Clifford circuits based on the methods proposed in <cite data-cite=\"schneider2023satEncodingOptimalClifford\">A SAT Encoding for Optimal Clifford Circuit Synthesis</cite>.\n",
```

Comment:
The bib-entry corresponding to the ASPDAC paper is currently missing, which is why this is not rendered correctly.
A corresponding statement ("If you use ..., please cite X") is also missing on the Publications page.